index.js

```diff
@@ -0,0 +1,2 @@
 strict";var r=t(79),a=t(3),i=a.publishing.uploadDirectory,o=r({uploadDir:i});e.exports=o},function(e,n){e.exports=require("connect-multiparty")},function(e,n,t){"use strict";var r=t(81),a=t(82),i=t(83);e.exports=function(e){e.get("/",r),e.get("/login",r),e.get("/about",r),e.get("/trending",i("/popular")),e.get("/popular",r),e.get("/new",r),e.get("/embed/:claimId/:name",a)}},function(e,n,t){"use strict";var r=t(14),a=function(e,n){r(e,n)};e.exports=a},function(e,n,t){"use strict";var r=t(3),a=r.details.host,i=function(e,n){var t=e.params,r=t.claimId,i=t.name;n.status(200).render("embed",{layout:"embed",host:a,claimId:r,name:i})};e.exports=i},function(e,n,t){"use strict";var r=function(e){return function(n,t){t.status(301).redirect(e)}};e.exports=r},function(e,n,t){"use strict";var r=t(85),a=t(88);e.exports=function(e,n){e.get("/:identifier/:claim",a),e.get("/:claim",r)}},function(e,n,t){"use strict";var r=t(4),a=r.sendGAServeEvent,i=t(23),o=i.determineResponseType,l=i.logRequestData,s=i.getClaimIdAndServeAsset,u=t(24),c=t(25),d=function(e,n){var t=e.headers,r=e.ip,i=e.originalUrl,d=e.params,f=void 0;try{f=u.parseModifier(d.claim).hasFileExtension}catch(e){return n.status(400).json({success:!1,message:e.message})}var h=o(f,t);if("SERVE"!==h)return c(e,n);a(t,r,i);var m=void 0;try{m=u.parseClaim(d.claim).claimName}catch(e){return n.status(400).json({success:!1,message:e.message})}l(h,m,null,null),s(null,null,m,null,i,r,n)};e.exports=d},function(e,n){e.exports=require("redux-saga")},function(e,n){e.exports=require("redux-saga/effects")},function(e,n,t){"use strict";var r=function(){function e(e,n){var t=[],r=!0,a=!1,i=void 0;try{for(var o,l=e[Symbol.iterator]();!(r=(o=l.next()).done)&&(t.push(o.value),!n||t.length!==n);r=!0);}catch(e){a=!0,i=e}finally{try{!r&&l.return&&l.return()}finally{if(a)throw i}}return t}return function(n,t){if(Array.isArray(n))return n;if(Symbol.iterator in Object(n))return e(n,t);throw new TypeError("Invalid attempt to destructure non-iterable instance")}}(),a=t(4),i=a.sendGAServeEvent,o=t(23),l=o.determineResponseType,s=o.flipClaimNameAndIdForBackwardsCompatibility,u=o.logRequestData,c=o.getClaimIdAndServeAsset,d=t(24),f=t(25),h=function(e,n){var t=e.headers,a=e.ip,o=e.originalUrl,h=e.params,m=void 0;try{m=d.parseModifier(h.claim).hasFileExtension}catch(e){return n.status(400).json({success:!1,message:e.message})}var p=l(m,t);if("SERVE"!==p)return f(e,n);i(t,a,o);var g=void 0;try{g=d.parseClaim(h.claim).claimName}catch(e){return n.status(400).json({success:!1,message:e.message})}var v=void 0,y=void 0,b=void 0,w=void 0;try{var C=d.parseIdentifier(h.identifier);v=C.isChannel,y=C.channelName,b=C.channelClaimId,w=C.claimId}catch(e){return n.status(400).json({success:!1,message:e.message})}if(!v){var I=s(w,g),x=r(I,2);w=x[0],g=x[1]}u(p,g,y,w),c(y,b,g,w,o,a,n)};e.exports=h},function(e,n,t){"use strict";var r=t(90);e.exports=function(e){e.get("*",r)}},function(e,n,t){"use strict";var r=t(14),a=function(e,n){r(e,n)};e.exports=a}]);
```

Comment:
Don't commit this

Reply:
hmm, the problem is that the server uses jsx for server side rendering, so it needs to be transpiled (I think that's the only part that needs to be transpiled), so I wanted to build a bundle and export that bundle.  Do you think I should just transpile with babel and not actual bundle with webpack?  That's what I did for [spee.ch-components](https://github.com/lbryio/spee.ch-components)

Reply:
Oh I gotcha. I was confused.

I still don't think this needs to be committed. You can build before publishing. IMO if someone is cloning the repo, they will be building the app on their own.

Reply:
i updated it to only transpile the two jsx render functions that are needed for SSR, the rest does not need to be built at this level, as the implementation will build the app again.